search/src/Components/wrapper/reactive_search.js

```diff
@@ -562,6 +563,16 @@ class ReactiveSearchComponent extends Component {
                 headers={{'X-CSRFToken': Cookies.get('csrftoken')}}
             >
                 <div style={{ position: "fixed", width: "20rem", overflow: "scroll", height: "90%", left: 0, padding: '0 1rem' }}>
+                    <MultiList
```

Comment:
I wonder whether this should appear in the results pane. From the user's perspective, the datasets that show up are results. OTOH, having something that appears like a filter appear on both the left and the right might be surprising. Your thoughts?

Reply:
Any thoughts on putting it in the right-side pane?